src/main/resources/db/reference/modernised_darts_standing_data.sql

```diff
@@ -557,6 +557,6 @@ INSERT INTO transcription_type (trt_id, description, display_state) VALUES (999,
 INSERT INTO transcription_urgency (tru_id, description, display_state) VALUES (1, 'Standard', false);
 INSERT INTO transcription_urgency (tru_id, description, display_state) VALUES (2, 'Overnight', true);
 INSERT INTO transcription_urgency (tru_id, description, display_state) VALUES (3, 'Other', false);
-INSERT INTO transcription_urgency (tru_id, description, display_state) VALUES (4, '3 working days', true);
-INSERT INTO transcription_urgency (tru_id, description, display_state) VALUES (5, '7 working days', true);
-INSERT INTO transcription_urgency (tru_id, description, display_state) VALUES (6, '12 working days', true);
+INSERT INTO transcription_urgency (tru_id, description, display_state) VALUES (4, 'Up to 3 working days', true);
```

Comment:
Please can you also update the `src/main/resources/db/reference/modernised_darts_standing_data.sql` to ensure this isn't lost later on if we ditch the developer migrations?

Reply:
@richardbull this is that exact file 😂